lib/driver.js

```diff
@@ -265,6 +265,7 @@ class AndroidUiautomator2Driver extends BaseDriver {
     this.caps.deviceUDID = this.opts.udid;
     this.caps.platformVersion = await this.adb.getPlatformVersion();
     this.caps.deviceScreenSize = await this.adb.getScreenSize();
+    this.caps.deviceScreenDensity = await this.adb.getScreenDensity();
```

Comment:
perhaps, the comment above should be updated

Reply:
Fixed and force pushed for a clean history ;)